installation/index.md

```diff
@@ -28,34 +29,30 @@ Always keep this in mind when searching for help and solutions.
 Many devices are suited to host a continuous installation of openHAB.
 Experiences with different devices and environments can be found in the [community forum hardware section](https://community.openhab.org/c/hardware/server).
 
-The Raspberry Pi as a minimal sufficient device is quite popular, especially as we offer a quick setup with [openHABian](openhabian.html).
-A popular alternative is [our solution for the Synology DiskStation](synology.html), which many users already own in their homes.
-The previously mentioned [openHABian](openhabian.html) can also be used to kickstart your openHAB experience on existing Debian/Ubuntu based Linux systems.
-
-Please check the menu to the left for all available options.
+We recommend to get a dedicated Raspberry Pi 4 and to use [openHABian](openhabian.html) for quick setup. openHABian is also available to kickstart your openHAB experience on existing Debian x86 based Linux systems.
 
 ## Prerequisites
 
 Make sure that you have an up to date **Java 11** JVM platform installed on your host system.
-Zulu is currently the recommended Java platform for openHAB.
-[Download](https://www.azul.com/downloads/zulu-community/?&architecture=x86-64-bit&package=jdk#) and [installation](https://docs.azul.com/zulu/zuludocs/ZuluUserGuide/InstallingZulu/InstallationWindowsUsingZuluMSIFile.htm) instructions can be found on Azul Systems' Zulu website.
-Oracle Java is also suitable for most configurations, but licensing restrictions may apply.
-OpenJDK may also be used, but it has some [known limitations](https://community.openhab.org/t/running-openhab-2-on-openjdk/21443/8?u=gatekeeper6838) with openHAB and is not recommended.
+OpenJDK is currently the recommended Java platform for openHAB.
```

Comment:
I think you mean the openjdk packages that come with your distro when applicable. All the listed JVMs are based on the [OpenJDK](https://openjdk.org/) project code so you might want to clarify this.

Reply:
Changed wording but the problem is we don't know who reads this and that it has to be consistent with other docs.
Even Windows users think it applies to them.

Reply:
Thanks! That would be nice if they already used Google. :wink: